extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -220,6 +228,19 @@ export class AmpAnalytics extends AMP.BaseElement {
     this.analyticsGroup_ =
         this.instrumentation_.createAnalyticsGroup(this.element);
 
+    if (this.config_['transport'] && this.config_['transport']['iframe']) {
+      Transport.processCrossDomainIframe(this.getAmpDoc().win.document,
+        this.config_['transport'],
+        (msg) => {
+          try {
+            if (this.element.ownerDocument.location.href !== 'about:srcdoc') {
```

Comment:
This is to allow the creative to communicate with the amp-analytics instance?  Surely this code is temporary as otherwise it would be XSS?  Add TODO/DO NOT SUBMIT?